oak_utils/src/lib.rs

```diff
@@ -163,6 +163,97 @@ impl prost_build::ServiceGenerator for OakServiceGenerator {
     }
 }
 
+struct AsyncServiceGenerator;
+
+impl prost_build::ServiceGenerator for AsyncServiceGenerator {
```

Comment:
Is it possible to add a simple example that uses this? It would be nice to see the output of the generator.

Reply:
I have a separate branch that modifies the chat example to use the async service generator: https://github.com/wildarch/oak/blob/async_service_with_chat/examples/chat/module/rust/src/lib.rs.

I would prefer not to add it to this PR yet, to keep the changes small and self-contained.

Reply:
Thanks for the link :)